experimental/LieAlgebras/src/WeylGroup.jl

```diff
@@ -597,6 +597,29 @@ end
 ###############################################################################
 # G-set functionality (can get moved to src/ once `isomorphism(PermGroup, ::WeylGroup)` gets moved)
 
+"""
+    natural_gset(W::WeylGroup)
+
+Return the G-set `Omega` that consists of the closure of root space elements
+under the natural action of `W`.
+
+!!! warning
+    Currently the seeds are the minimal roots of the root system.
+    This may only be sane for finite types and there may be changes for infinite root systems in the future.
```

Comment:
```suggestion
!!! note
    This function is only implemented for finite Weyl groups, that is root systems of classical type.
```
this no longer fits the function